### PR TITLE
Skip windows build

### PR DESCRIPTION
**Pull request by ocefpaf**
Not ready yet...

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -37,14 +37,14 @@ before_install:
...
     - python bootstrap-obvious-ci-and-miniconda.py ~/miniconda x64 3 --without-obvci && source ~/miniconda/bin/activate root
     - conda config --set show_channel_urls True
     - conda install -c http://conda.binstar.org/pelson/channel/development --yes --quiet obvious-ci
     - conda config --add channels ioos
     - obvci_install_conda_build_tools.py
-    - conda install --yes conda-build==1.18.0
 
 script:
-    - obvci_conda_build_dir.py ./ ioos --channel main --build-condition "python >=2.7,<3|==3.4"
+    - obvci_conda_build_dir.py ./ SciTools --channel main --build-condition "numpy >=1.9" "python >=2.7,<3|>=3.4,<3.5"
```
Comment (pelson) [inline .travis.yml]: Woops.

Comment (ocefpaf) [inline .travis.yml]: OhOh!! Copy-and-paste :hankey:

No biggie.  This PR is far from ready :wink:

File: compliance-checker/meta.yaml [hunk 2]
```diff
@@ -26,7 +22,7 @@ requirements:
         - requests
         - python-dateutil
     run:
-        - python >=2.7,<3
```
Comment (pelson) [inline compliance-checker/meta.yaml]: You've lost the fact that this is py2 only?

File: iris/meta.yaml [hunk 3]
```diff
@@ -20,18 +16,22 @@ requirements:
         - biggus
         - cartopy
         - netcdf4
+        - numpy x.x
         - udunits2
         - pyke
         - setuptools
         - libmo_unpack  # [not win]
     run:
-        - python >=2.7,<3
```
Comment (pelson) [inline iris/meta.yaml]: Same as above. Is this in the build requirements in stead perhaps?

File: compliance-checker/meta.yaml (hunk 2 above)
Comment (ocefpaf) [inline compliance-checker/meta.yaml]: But I thought that with the new modification we should constraint the python version at the `build` filed only.

**Comment by ocefpaf** [comment]
Some lessons learned from this PR:

- Somewhere between `conda-build==1.18.0` and the latest `conda` version `conda-build` started choking on empty fields. Since we need the latest `conda` for this PR we also need to remove the empty fields. One might argue that we must to fix `conda-build` instead. However, that bug forces us to write cleaner `meta.yaml` :wink: (Changes: no longer downgrade `conda-build` and removed empty fields.)

- For some weird reason I could not get this PR working with miniconda3 on Travis-CI (OSX). That is not a problem because the encoding issue is finally fixed upstream. We are OK using miniconda2.

- In this PR I re-introduced the numpy build strings. (See https://github.com/ioos/conda-recipes/issues/507)

File: compliance-checker/meta.yaml (hunk 2 above)
Comment (pelson) [inline compliance-checker/meta.yaml]: Ah. I see. My bad.

**Comment by ocefpaf** [comment]
The AppVeyor failure is just a time-out.  It seems that this PR is finally good to go :tada: 

Thanks @pelson for the new goodies in `obvious-ci`!

**Pull request merged**